.vsts-ci.yml

```diff
@@ -46,6 +46,13 @@ stages:
 - stage: build
   jobs:
   - job: Publish_Build_Configuration
+    pool:      
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        name: NetCore1ESPool-Svc-Public
```

Comment:
@marcpopMSFT since this is main, the pools should be `NetCore1ESPool-public` and `NetCore1ESPool-Internal`

You should definitely use the `-Svc-` pools for all servicing branches though.

Reply:
what @akoeplinger said